### PR TITLE
Fix issue #214: Feature: include all the previous conversation history in the github issue/PR

### LINKED ISSUE #214: Feature: include all the previous conversation history in the github issue/PR
Instead of the issue/PR description, we should also include all the previous conversations in the issue thread and pass it to the agent in `resolve_issues.py`

### PR DESCRIPTION
**Pull request by openhands-agent**
This pull request fixes #214.

The PR successfully addresses the requirement to include all previous conversations from issue threads in the agent's context. Here's what was implemented:

Automatic fix generated by [OpenHands](https://github.com/All-Hands-AI/OpenHands/) 🙌

### CODE REVIEW

File: openhands_resolver/issue_definitions.py [hunk 1]
```diff
@@ -90,23 +113,66 @@ def get_converted_issues(self) -> list[GithubIssue]:
             if "pull_request" in issue:
                 continue
             
+            # Get issue thread comments
+            thread_comments = self._get_issue_comments(issue["number"])
+
```
Comment (neubig) [inline openhands_resolver/issue_definitions.py:118]: Please make sure that we have tests that test this case appropriately.

**Review by neubig — COMMENTED** [review]
Please add a test

**Comment by openhands-agent** [comment]
There seems to be incomplete information provided - the changes list cuts off after mentioning "two key tests in place" without specifying what those tests are or what other changes were made by OpenHands. Without seeing the complete list of changes and what issues they were meant to address, I cannot provide an accurate assessment of whether all issues have been successfully resolved. Please provide the full list of changes and the original issues they were meant to fix.

File: openhands_resolver/issue_definitions.py (hunk 1 above)
Comment (openhands-agent) [reply openhands_resolver/issue_definitions.py:118]: Openhands fix success summary


The AI agent has confirmed that appropriate test coverage exists for the thread comments functionality. Specifically, there are two key tests in place:

**Pull request merged**